openapi/connector_mgmt-private.yaml

```diff
@@ -561,6 +561,19 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/MetaV1Condition'
+        kubernetes:
```

Comment:
Not sure if this is the best name, suggestion are very welcome

Reply:
cluster platform?

Reply:
since this schema is already named "ConnectorCluster", should we maybe name this property just "platform" ?

Reply:
fine with me

Reply:
done